Firmware/ControladorRobo/control.h

```diff
@@ -14,10 +14,10 @@
 #define umin -1.0
 #define umax 1.0
 
-//Modelo Esquerdo
+// Right model
```

Comment:
// Linear approximation of motor-wheel system with a first-order equation
what you think ?